Build/SaveLoadGame.cc

```diff
@@ -2095,6 +2135,9 @@ static void LoadGeneralInfo(HWFILE const f, UINT32 const savegame_version)
 	EXTR_BOOL( d, gubPlayerProgressSkyriderLastCommentedOn)
 	EXTR_BOOL( d, gfMeanwhileTryingToStart)
 	EXTR_BOOL( d, gfInMeanwhile)
+	// Always set gfInMeanwhile to false for Dead is Dead. This must be done because it is saved as true if a Meanwhile event is in the event pipe
+	// Preventing the value to be saved in the first place leads to odd behaviour during the commencing cutscene
```

Comment:
Yes. At the moment saving within a cutscene causes the game to bug out if loading again.
E.g. in Deidrannas palace the roofs get opaque and you cannot see inside.
In the worst case the whole cutscene gets stuck forever.
This is why I prevent saving during a meanwhile at the moment, but the gfInMeanwhile variable is behaving quite odd in many cases. We should investigate further what the variable is actually doing and how it is used.
For the moment I am too afraid to break the rest of the game though and this workaround seems to have no side for the mode yet.